Source/WebKit/WebProcess/Inspector/WebInspectorClient.cpp

```diff
@@ -80,7 +80,7 @@ WebInspectorClient::~WebInspectorClient()
     if (m_paintRectOverlay) {
         RefPtr page = m_page.get();
         if (page && page->corePage())
-            page->corePage()->pageOverlayController().uninstallPageOverlay(*m_paintRectOverlay, PageOverlay::FadeMode::Fade);
+            page->corePage()->pageOverlayController().uninstallPageOverlay(*(m_paintRectOverlay.copyRef()), PageOverlay::FadeMode::Fade);
```

Comment:
You should obtain a `RefPtr` to `m_paintRectOverlay` as part of the conditional above and deref that here.